docs/how-to-use/search/advanced-search-user-docs.md

```diff
@@ -0,0 +1,65 @@
+# Advanced Search Component
+
+## Introduction
+
+The `AdvancedSearchComponent` allows for the creation of complex queries using a graphical widget.
+The widget's contents are then turned into a string representing a Gravsearch (SPAQRL) query to be sent to DSP-API.
+
+A query consists of the following elements:
+- ontology (data model) selection
+- selection of a resource class belonging to the selected ontology (optional)
+- specification of properties, comparison operators, and values (optional).
+
+Although selection of a resource or a property ore both are optional, either a resource class has to be selected or at least one property has to be specified,
+otherwise the query is not considered valid and cannot be submitted.
+
+## Comparison Operators
+
+Depending on the value type of the chosen property,
+one or more of the following comparison operators can be selected:
+
+- `is equal to`: value equality: same number, exact same string, overlap of date periods, same target resource.
```

Comment:
"matching date periods" is probably more clear here (same for the instances below), unless you truly do mean an overlap

Reply:
The thing is that each date is represented as a period in DSP-API. All kinds of overlap count as a match, e.g., a search for a date with day precision will match a value with year precision.

Reply:
That's fair. It's fine the way it is then :)

Reply:
I understand why you asked though.

Maybe it could be covered in more detail in the DSP-API docs and could be linked to from here?

Reply:
https://docs-api.dasch.swiss/03-apis/api-v2/query-language/#date-comparisons

Reply:
However, I am not sure how many technical details a user of the GUI should know